test/jdk/java/foreign/virtual/TestVirtualCalls.java

```diff
@@ -57,10 +55,9 @@
                 FunctionDescriptor.of(C_INT));
 
         System.loadLibrary("Virtual");
-        SymbolLookup lookup = SymbolLookup.loaderLookup();
-        funcA = lookup.lookup("funcA").get();
-        funcB = lookup.lookup("funcB").get();
-        funcC = lookup.lookup("funcC").get();
+        funcA = TestVirtualCalls.class.getClassLoader().findNative("funcA").get();
+        funcB = TestVirtualCalls.class.getClassLoader().findNative("funcB").get();
+        funcC = TestVirtualCalls.class.getClassLoader().findNative("funcC").get();
```

Comment:
Maybe a helper method could be added to `NativeTestHelper`, look `findNativeOrThrow(...)` to make lookups a little bit simpler in the tests

Reply:
good idea, I'll add that.